podcasts/PodcastViewController.swift

```diff
@@ -829,7 +829,7 @@ class PodcastViewController: FakeNavViewController, PodcastActionsDelegate, Sync
     }
 
     func fundingTapped() {
-        Analytics.track(.podcastScreenFundingTapped)
+        Analytics.track(.podcastScreenFundingTapped, properties: ["uuid": podcast?.uuid ?? ""])
```

Comment:
Isn't this adding `uuid` and not `podcast_uuid`?

Reply:
Correct, needs to be updated to `podcast_uuid` to match Android

Reply:
It took me a few to realize you're talking about the property name and not the value. I started looking for podcast.podcast_uuid, checked the value of the uuid etc. I updated the property name to podcast_uuid.